exercises/concept/vehicle-purchase/vehicle-purchase.awk

```diff
@@ -0,0 +1,6 @@
+# Task 1: Print true if the first field is "needs_license"
+#         and the second field contains "car" or "truck".
+
+# Task 2: When the first field is "resell_price",
```

Comment:
I like the "When the first field" phrasing better than the "... if the first field is" in task one. Can we can that for both?